e2e/react-start/rsc/src/routes/rsc-stream-readable.tsx

```
@@ -0,0 +1,279 @@
+import * as React from 'react'
+import { createFileRoute, useHydrated } from '@tanstack/react-router'
+import { CompositeComponent } from '@tanstack/react-start/rsc'
+import { streamNotificationsReadable } from '~/utils/streamingServerComponents'
+import {
+  pageStyles,
+  clientStyles,
```

ESLint

[error] Member 'clientStyles' of the import declaration should be sorted alphabetically.

(sort-imports)